addon/components/plugins/indentation/indentation-menu.ts

```diff
@@ -18,11 +20,25 @@ export default class IndentationMenuComponent extends Component<Args> {
     return this.controller.schema;
   }
 
+  get extraTypes() {
+    if (!this.args.extraTypes) return [];
+    if (this.args.extraTypes instanceof Array) return this.args.extraTypes;
+    return [this.args.extraTypes];
+  }
+
+  get allowedTypes() {
```

Comment:
I would modify this so that all the indentable types are overriden by `this.args.types` and by default use `[this.schema.nodes.paragraph, this.schema.nodes.heading]`. In this way, it is fully customizable.

Reply:
Yes, that makes more sense :+1: